fgd/point/prop/prop_static.fgd

```diff
@@ -138,7 +138,7 @@
 
 	linedivider_fade[!engine](string) readonly : "----------------------------------------------------------------------------------------------------------" : ""
 
-	screenspacefade[until_ASW](boolean) : "Screen Space Fade" : 0 : "The method by which the fading distance should be determined. If 'No', the fade distances is the distance from the player's view to the object, in inches. " +
```

Comment:
And for this, does `until_ASW` mean that ASW+ (including Chaos) *doesn't* have this?

Reply:
Again, I split up the PRs mainly for ease of review. The 11 should be viewed as one big change.
This would be viewed as not in chaos, so it is removed. This field is *technically* in chaos, but marked as obsolete so i opted to remove it.